src/main/java/com/bigchaindb/util/JsonUtils.java

```diff
@@ -35,6 +35,7 @@
 
     private static Map<String, TypeAdapter> typeAdaptersSerialize = new ConcurrentHashMap<String, TypeAdapter>(16) {{
         put(Asset.class.getCanonicalName(), new TypeAdapter(Asset.class, new AssetSerializer()));
+        put(Input.class.getCanonicalName(), new TypeAdapter(Input.class, new InputSerializer()));
```

Comment:
here is where we mount the new `InputSerializer`

Reply:
accepted